src/virt.c

```diff
@@ -715,9 +762,13 @@ refresh_lists (void)
             xpath_ctx = xmlXPathNewContext (xml_doc);
 
             /* Block devices. */
-            xpath_obj = xmlXPathEval
-                ((xmlChar *) "/domain/devices/disk/target[@dev]",
-                 xpath_ctx);
+	    char * bd_xmlpath = "/domain/devices/disk/target[@dev]";  //the default behavior
```

Comment:
That comment states the obvious, please remove. Also, please use a consistent style, i.e. `char *name` rather than `char * name`.